api/v1beta1/openstackdataplanedeployment_types.go

```diff
@@ -42,6 +42,10 @@ type OpenStackDataPlaneDeploymentSpec struct {
 	// +kubebuilder:validation:Optional
 	AnsibleSkipTags string `json:"ansibleSkipTags,omitempty"`
 
+    // +kubebuilder:validation:Optional
+    // AnsibleVars for ansible execution
+    AnsibleVars []string `json:"ansibleVars,omitempty"`
```

Comment:
```suggestion
    // +kubebuilder:validation:Optional
    // AnsibleVars for ansible execution
    // +kubebuilder:pruning:PreserveUnknownFields
    // +kubebuilder:validation:Schemaless
    AnsibleVars map[string]json.RawMessage `json:"ansibleVars,omitempty"`
```
https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/api/v1beta1/common.go#L45